sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector_node.hpp

```diff
@@ -52,6 +52,9 @@ class DistortionCorrectorComponent : public rclcpp::Node
   std::string base_frame_;
   bool use_imu_;
   bool use_3d_distortion_correction_;
+  bool update_azimuth_and_distance_;
+
+  bool can_update_azimuth_and_distance_{false};
```

Comment:
Does this need to be in `distortion_corrector_node.hpp` as opposed to `distortion_corrector.hpp`? If it was in the latter, it could be combined with the struct for `a_` and `b_` into an `std::optional<AngleConversion>`.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/13d981f70a5fa22fd07a8f7dfa302420a1700a6e, https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/8a24a79457c1204a21b0a398d90a4ad5185ea82e, https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/02b12177da2092ae663a3c27b0bccca07a3876c2